client/src/components/pages/profile/profile-tasks/side-bar.ts/side-bar.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+
+interface ITasksSideBar {
+	items: {
+		id: string;
+		title: string;
+		count: number;
+	}[];
+	onClick: (key: number) => void;
+}
+
+export const ProfileTasksSideBar: React.FC<ITasksSideBar> = ({ items, onClick }) => {
+	const [activeItem, setActiveItem] = useState<number>(0);
```

Comment:
Better to use state for active item id instead of index. It is better to put this state in a container so that it is convenient to organize the logic. Since when switching tabs, you will need to display different information